Sources/OpenAPIURLSession/URLSessionTransport.swift

```diff
@@ -178,6 +178,8 @@ extension URLSessionTransportError: CustomStringConvertible {
                 "Invalid request URL from request path: \(request.path), query: \(request.query ?? "<nil>") relative to base URL: \(baseURL.absoluteString)"
         case .notHTTPResponse(let response):
             return "Received a non-HTTP response, of type: \(String(describing: type(of: response)))"
+        case .noResponse(let url):
+            return "Received a nil response for \(url?.absoluteString ?? "")"
```

Comment:
```suggestion
            return "Received a nil response for \(url?.absoluteString ?? "<nil URL>")"
```

Nit: to make it easier to catch when the URL is nil.